src/cljs/nr/cardbrowser.cljs

```diff
@@ -26,21 +28,58 @@
           sets (:json (<! (GET "/data/sets")))
           cycles (:json (<! (GET "/data/cycles")))
           mwls (:json (<! (GET "/data/mwl")))
-          latest_mwl (->> mwls
+          latest-mwl (->> mwls
                           (filter #(= "standard" (:format %)))
                           (map (fn [e] (update e :date-start #(js/Date.parse %))))
                           (sort-by :date-start)
-                          last)]
-      (reset! cards/mwl latest_mwl)
+                          last)
+            alt-cards (->> cards
+                       (map #(select-keys % [:title :code :alt_art]))
+                       (filter :alt_art)
+                       (into {} (map (juxt :code identity))))
```

Comment:
I don't think this line does what you intend. The `->>` will transform it to be `(into {} (map (juxt :code identity)) (result-of-previous-lines))`. Meaning the `map` call will create a transducer and then be fed into the hashmap, along with the output of the previous lines. Maybe you meant: `(->> ... (filter :alt_art) (map (juxt :code identity)) (into {}))`?